parser/distsql/statement/src/main/java/org/apache/shardingsphere/distsql/statement/ral/updatable/UnlockClusterStatement.java

```diff
@@ -19,9 +19,16 @@
 
 import lombok.RequiredArgsConstructor;
 
+import java.util.Optional;
```

Comment:
The checkstyle break here, please keep blank line.

Reply:
Thanks for your reminding, I have fixed it